src/store/modules/user/actions.ts

```diff
@@ -58,6 +58,15 @@ const actions: ActionTree<UserState, RootState> = {
       if (resp.data.userTimeZone !== localTimeZone) {
         emitter.emit('timeZoneDifferent', { profileTimeZone: resp.data.userTimeZone, localTimeZone});
       }
+      const userPreferenceResp = await UserService.getUserPreference({
+        'userPrefTypeId': 'BOPIS_PREFERENCE'
+      });
+
+      if (userPreferenceResp.status == 200 && !hasError(userPreferenceResp) && userPreferenceResp.data?.userPrefValue) {
+        const userPreference = JSON.parse(userPreferenceResp.data.userPrefValue)
+        commit(types.USER_PREFERENCE_UPDATED, userPreference.shippingOrderStatus)
```

Comment:
Pass entire preference received as we are updating the preference and not the showShippingOrders

Reply:
Made the requested change.